public/css/app-login.css

```diff
@@ -0,0 +1,44 @@
+body {
+    background: url('../img/stats-background.jpg') no-repeat center center fixed;
+    -webkit-background-size: cover;
+    -moz-background-size: cover;
+    -o-background-size: cover;
+    background-size: cover;
+}
+
+.register, .login {
+    background: url('../img/login-background.jpg') no-repeat center center fixed;
+    -webkit-background-size: cover;
+    -moz-background-size: cover;
+    -o-background-size: cover;
+    background-size: cover;
+}
+
+.register h2, .login h2 {
+    margin-top: inherit;
+}
+
+.panel-default {
+    background-color: rgba(255, 255, 255, 0.6)
+}
+
+.register #messages, .login #messages {
+    padding-top: 30px;
+    float: left;
+}
+
+.top-buffer-auth {
+    margin-top: 180px;
```

Comment:
This one change doesn't warrant making almost completely identical CSS and layout template files. `top-buffer-auth` didn't used to exist before I started making the dashboard page, it was broken into two css classes because the pages had different display requirements. We can do something similar to make the login page have a different top margin than the registration page.